scripts/tei.p5.xsl

```diff
@@ -36,6 +36,14 @@
 <!-- ==================================================================== -->
 <!--                            OVERRIDES                                 -->
 <!-- ==================================================================== -->
+  
+  <!-- Adding links to other cases via bibls https://github.com/CDRH/earlywashingtondc/issues/124 -->
+  
+  <xsl:template match="/TEI/text//bibl[@type='case'][@xml:id]">
+    <a href="{@xml:id}" class="tei_bibl_link">
```

Comment:
Does this work with relative links to the cases?  Or should we be subbing in a site-url for the first portion of the href?

Reply:
It would still work even if the URL changes. It's relative, not root.

Reply:
Tested locally and it works. the doc links are also relative.